bitnami/redis-cluster/templates/update-cluster.yaml

```diff
@@ -90,13 +92,29 @@ spec:
                   echo "Node $nodeIp not ready, waiting for all the nodes to be ready..."
                   sleep 5
                 done
+                slave=()
+                if (( new_node_index % 2 )); then
+                  slave+=("--cluster-slave")
+                  [ -z previousNodeIp ] && previousNodeIp=$firstNodeIp
```

Comment:
I think `firstNodeIp` and `slave` will not be defined here.
Apart from that, if we always use first node IP when searching the master, won't the salves be always added to the same master? i.e. the one with index `0`.

Reply:
I am fixing, previousNodeIP, but slave needs to be reset in each loop iteration.